x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/helpers.ts

```diff
@@ -161,6 +164,30 @@ export const filterRuleFieldsForType = <T extends Partial<RuleFields>>(
   assertUnreachable(type);
 };
 
+function filterThreatByName<T extends IMitreAttack | IMitreAttackTechnique>(filterable: T[]): T[] {
```

Comment:
I think we could rename this function to better express what it does. When I read `filterThreatByName` I expect the function to take a `name` parameter an return threats matching that name.